nitransforms/io/itk.py

```diff
@@ -196,14 +184,25 @@ def to_string(self):
         strings = []
         for i, xfm in enumerate(self.xforms):
             xfm.structarr['index'] = i
-            strings.append(xfm.to_string())
+            strings.append(xfm.to_string(banner=(i == 0)))
```

Comment:
this method is missing the `banner` parameter

Reply:
what do you mean? Isn't it defined on line 72?

Reply:
ah, discard that - sorry for the noise